server/handlers/meshery_pattern_handler.go

```diff
@@ -2069,7 +2070,7 @@ func createArtifactHubPkg(pattern *models.MesheryPattern, user string) ([]byte,
 	if isCatalogItem {
 		version = pattern.CatalogData.PublishedVersion
 	}
-	artifactHubPkg := catalog.BuildArtifactHubPkg(pattern.Name, "", user, version, pattern.CreatedAt.String(), &pattern.CatalogData)
+	artifactHubPkg := catalog.BuildArtifactHubPkg(pattern.Name, "", user, version, pattern.CreatedAt.Format(time.RFC3339), &pattern.CatalogData)
```

Comment:
Thanks for this @ctcarrier . 

Ideally, we would want this to be handled within the `BuildArtifactHubPkg` in Meshkit. The `BuildArtifactHubPkg` function should take `time` type as a parameter rather than `string`. And inside the `BuildArtifactHubPkg` function, we would try to parse it to RFC3339 and return if it failed.

Reply:
New issue created to track this enhancement - https://github.com/meshery/meshkit/issues/556